HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,142 @@
+
+```
+### Graph Fail 1 (CORRECTED)
+# first argument of ggplot is the dataset, 'city' variable is actually cty, and to add a geom_point layer, use a '+'.
+```{r}
+data(mpg) #this is a dataset from the ggplot2 package
+
+mpg %>% 
+  ggplot(data = mpg, mapping = aes(x = cty, y = hwy, color = "blue")) +
+  geom_point()
+
+```
+
+
+
+
+### Graph Fail 2
+Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+```{r}
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = "blue"))
+
+```
+### Graph Fail 2 (CORRECTED)
+# color goes outside of aes because it is not conveying info about the data.
+```{r}
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
+
+```
+
+
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(5, 40))
+
+```
+### Graph Fail 3 (CORRECTED)
+# alpha values range from 0 to 1, or transparent to opaque, modifying the transparency of the data points. If you want the points to be be slightly bigger you should set shape to a higher value instead.
+#You can change legend location through the theme elements, specifying general sections of the graph or specific coordinates. In order to remove the legend completely you would use the following layer: 'theme(legend.position = "none")
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = (c(5, 40)) +
+  theme(legend.title = element_blank())
+
+```
+
+
+
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy, color = drv)) + 
+  geom_point() + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```
+### Graph Fail 4 (CORRECTED)
+#any aesthetics added to the ggplot applies to every subsequent layer. Moving the 'color = drv' to the geom_point layer allows for the desired data representation.
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point(aes, color = drv) + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
```

Comment:
This makes more sense than what I ended up doing to get one smoothing line. I added a "group" argument within ggplot(aes()) but this code looks cleaner.